data/scripts/spells/cancel_magic_shield.lua

```diff
@@ -0,0 +1,26 @@
+if configManager.getBoolean(configKeys.MANASHIELD_BREAKABLE) == true then
```

Comment:
just check `configManager.getBoolean(configKeys.MANASHIELD_BREAKABLE)` without `== true` the same in the other places where you did this